src/pages/Admin.jsx

```diff
@@ -8,8 +8,8 @@ const Home = () => {
         display: "flex",
         flexWrap: "wrap",
         gap: 6,
-        height: "70vh",
-        width: "70vw",
+        height: "100vh",
+        width: "100vw",
```

Comment:
not sure if this change was necessary by actual viewing of the buttons original positions on the screen